### PR TITLE
fix: Plugin initializers onActivate onDeactivate

### PR DESCRIPTION
**Pull request by remisultan**
fixes https://github.com/gravitee-io/issues/issues/6942

### CODE REVIEW

**Comment by remisultan** [comment]
Still need to discuss this while this is in draft.
@tcompiegne @leleueri

File: gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/config/gravitee.yml [hunk 1]
```diff
@@ -276,6 +276,19 @@ gateway:
 #  retryInterval: 1000 # delay in ms between two retries (default: 1000)
 #  expirationTimeOut: 300 # the end of validity of the data in seconds (default 300 = 5 minutes)
 
+groovy:
```
Comment (leleueri) [inline gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/config/gravitee.yml:279]: should it be part of the commit ?

File: gravitee-am-management-api/gravitee-am-management-api-standalone/gravitee-am-management-api-standalone-distribution/src/main/resources/config/gravitee.yml [hunk 2]
```diff
@@ -278,7 +278,14 @@ user:
 gateway:
   url: http://localhost:8092
 
-# global configuration of the http client
+groovy:
```
Comment (leleueri) [inline gravitee-am-management-api/gravitee-am-management-api-standalone/gravitee-am-management-api-standalone-distribution/src/main/resources/config/gravitee.yml:281]: should it be part of the commit ?

**Review by leleueri — CHANGES_REQUESTED** [review]

File: gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/config/gravitee.yml (hunk 1 above)
Comment (remisultan) [reply gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/config/gravitee.yml:279]: good call

File: gravitee-am-management-api/gravitee-am-management-api-standalone/gravitee-am-management-api-standalone-distribution/src/main/resources/config/gravitee.yml (hunk 2 above)
Comment (remisultan) [reply gravitee-am-management-api/gravitee-am-management-api-standalone/gravitee-am-management-api-standalone-distribution/src/main/resources/config/gravitee.yml:281]: good call

**Review by tcompiegne — APPROVED** [review]

**Pull request merged**